app/views/network_router/_common_new_edit.haml

```diff
@@ -0,0 +1,71 @@
+%h3
+  = _('Basic Information')
+.form-horizontal
+  .form-group
+    %label.col-md-2.control-label
+      = _('Router Name')
+    .col-md-8
+      %input.form-control{:type          => "text",
+                          :name          => "name",
+                          'ng-model'     => "networkRouterModel.name",
+                          'ng-maxlength' => 128,
+                          :miqrequired   => true,
+                          :checkchange   => true}
+%h3
```

Comment:
@gildub Some fields are `Required`, because otherwise the `Add` button will not light up (turn blue, that is) as seen in the screenshots below --

<img width="1150" alt="screen shot 2017-03-17 at 4 39 33 pm" src="https://cloud.githubusercontent.com/assets/1538216/24066518/9a8409ae-0b30-11e7-9d0c-41a621e1a13b.png">

<img width="1147" alt="screen shot 2017-03-17 at 4 44 37 pm" src="https://cloud.githubusercontent.com/assets/1538216/24066556/0dedc858-0b31-11e7-8774-254364e207a9.png">


We need to clearly say `Required` for every Required field, it just makes for a better UX, that's all.